pycm/pycm_handler.py

```diff
@@ -240,7 +240,7 @@ def __obj_file_handler__(cm, file):
     return matrix_param
 
 
-def __obj_matrix_handler__(matrix, transpose):
+def __obj_matrix_handler__(matrix, transpose, classes=None):
```

Comment:
We should try to follow one order of parameters in similar functions.
I suggest changing here to `classes=None, transpose`.